src/gc/pal/CMakeLists.txt

```diff
@@ -0,0 +1,10 @@
+set(CMAKE_INCLUDE_CURRENT_DIR ON)
```

Comment:
The Windows and Unix specific files should be symmetric. If you have Unix specific files under pal directory, the Windows specific files should be there too.